src/frontend/Komponenter/Oppgavebenk/typer/oppgaverequest.ts

```diff
@@ -17,4 +17,5 @@ export interface IOppgaveRequest {
     fristTom?: string;
     erUtenMappe?: boolean;
     ident?: string;
+    komponentKey?: string;
```

Comment:
Foreslår at vi legger til en liten kommentar om hvorfor dette feltet finnes. Slik at man ser at dette ikker er noe som sendes med fra eller brukes av backend:
```suggestion
    komponentKey?: string; // For intern re-rendering i frontend
```

Reply:
Enig! 💪